form-submit.html

```diff
@@ -0,0 +1,193 @@
+          <div class="font-weight-bold text-white h2 pl-1">SigLens</div>
+        </div>
+      </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="https://siglens.github.io/siglens-docs/" target="_blank">Documentation</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/blog/allblogs.html">Blogs</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/press-news.html">Press & News</a>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
+              aria-haspopup="true" aria-expanded="false"> Community </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <a class="dropdown-item" href="https://github.com/siglens/siglens">GitHub</a>
+              <a class="dropdown-item" href="/slack.html">Slack</a>
+              <a class="dropdown-item" href="/guides.html">Guides</a>
+              <a class="dropdown-item" href="/questions.html">Questions</a>
+            </div>
+          </li>
+          <li>
+            <a href="./index.html#getting-started" id="try-now-btn">Try Now</a>
+          </li>
+          <li>
+            <a href="/slack.html" target="_blank">
+              <img src="assets/slack.svg" width="22" height="22" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/siglens/siglens" target="_blank">
+              <img src="assets/github.svg" width="24" height="24" alt="github-logo" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <!-- Landing Section -->
+  <section class="container text-white my-md-5 my-0 h-50 position-relative pt-5 pt-md-0">
+    <img src="./assets/main-gradient.svg" alt="gradient" class="light-gradient position-absolute">
+    <ul class="hero-btn-container d-flex justify-content-start ml-md-5 pl-md-5 mt-md-5 pt-md-5 align-items-start ">
+      <div>
+        <li class="hero-btn">
+          <a href="/index.html" id="install-siglens-btn"><i class="fa fa-arrow-left mr-2" aria-hidden="true"></i>Go
+            Back</a>
+        </li>
+      </div>
+    </ul>
+    <div class="d-flex justify-content-center align-items-center flex-column mt-5 pt-md-2 ">
+      <div class="thank-tick d-flex align-items-center justify-content-center ">
+        <i class="fa-solid fa-check text-white"></i>
+      </div>
+      <h1 class="main-heading text-center text-uppercase mt-5">
```

Comment:
this font looks a bit big. Also do we want to say that "We will get in touch"

Reply:
Is this okay?
<img width="1431" alt="image" src="https://github.com/siglens/siglens.github.io/assets/71771131/cdead729-63b5-4316-9891-b38660ed1357">